geometry_msgs/Pose head
geometry_msgs/Pose left_hand
geometry_msgs/Pose right_hand
